axum-macros/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+use quote::{quote, ToTokens};
+use syn::parse::Parse;
+
+mod from_request;
+
+#[proc_macro_derive(FromRequest, attributes(from_request))]
+pub fn derive_from_request(item: proc_macro::TokenStream) -> proc_macro::TokenStream {
+    expand_with(item, from_request::expand)
+}
+
+fn expand_with<F, T, K>(input: proc_macro::TokenStream, f: F) -> proc_macro::TokenStream
```

Comment:
What's the point of this abstraction?

Reply:
Just removes some of the boilerplate. I expect this crate to get more macros overtime so I think it's nice. axum-debug could be moved in here.